core/api/graphql/schemas/commit.hash.txt

```diff
@@ -1 +1 @@
-bbaa54a830c30a5a77c104cbc3bb2c69c16667d5
+884c5c89d10b08a4e5c7a038dd00150b4bccd448
```

Comment:
Currently, this uses a dev commit (off a branch). Needs to be updated to a commit on master once the backend PR is merged.